From 8076c8d3e433a6f6859f990dfd11095ba7c01d78 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Mon, 17 Feb 2025 11:44:10 +0100
Subject: [PATCH 781/954] fix(st-clock): remove TODO in STM32MP2 clock driver

Remove unnecessary TODO comments.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git f95edc73576725e15122741b6965469011b777c6 fix(st-clock): remove TODO in STM32MP2 clock driver]
Change-Id: I23203dfa3eb465735459e345d19b7f89a4310cfe
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/437584
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 drivers/st/clk/clk-stm32mp2.c | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/drivers/st/clk/clk-stm32mp2.c b/drivers/st/clk/clk-stm32mp2.c
index 86f8d8a306..c8917d770f 100644
--- a/drivers/st/clk/clk-stm32mp2.c
+++ b/drivers/st/clk/clk-stm32mp2.c
@@ -492,7 +492,6 @@ enum enum_gate_cfg {
 	.set_clr	= (_offset_clr),\
 }
 
-/* TODO : ADD MCO... ETH */
 static const struct gate_cfg gates_mp2[LAST_GATE] = {
 	GATE_CFG(GATE_LSE,		RCC_BDCR,		0,	0),
 #if STM32MP21
@@ -1843,7 +1842,6 @@ static void stm32mp2_clk_xbar_on_hsi(struct stm32_clk_priv *priv)
 #endif
 #endif /* IMAGE_BL2 */
 
-/* TODO: MOVE THIS FUNCTION A35 ONLY */
 static int stm32mp2_a35_pll1_start(void)
 {
 	uintptr_t a35_ss_address = A35SSC_BASE;
@@ -2029,11 +2027,6 @@ static int _clk_stm32_pll1_init(struct stm32_clk_priv *priv, int pll_idx,
 	unsigned long refclk;
 	int ret = 0;
 
-	/*
-	 * TODO: check if pll has already good parameters or if we could make
-	    a configuration on the fly.
-	 */
-
 	stm32mp2_a35_ss_on_bypass();
 
 #if STM32MP_M33_TDCID
@@ -2092,11 +2085,6 @@ static int _clk_stm32_pll_init(struct stm32_clk_priv *priv, int pll_idx,
 	bool spread_spectrum = false;
 	int ret = 0;
 
-	/*
-	 * TODO: check if pll has already good parameters or if we could make
-	    a configuration on the fly.
-	 */
-
 	_clk_stm32_pll_disable(priv, pll);
 
 	ret = stm32_clk_configure_mux(priv, pll_conf->src);
@@ -2311,7 +2299,6 @@ static int stm32mp2_clk_flexgen_configure(struct stm32_clk_priv *priv)
 		pdiv = (cmd_data & FLEX_PDIV_MASK) >> FLEX_PDIV_SHIFT;
 		fdiv = (cmd_data & FLEX_FDIV_MASK) >> FLEX_FDIV_SHIFT;
 
-		/* TODO: check if channel can be reconfigured */
 		switch (channel) {
 		case 33U: /* STGEN */
 			break;
-- 
2.53.0

